src/filesystem/file_content.rs

```diff
@@ -150,10 +164,15 @@ impl FileContent {
         matches!(self, Self::Stub { .. })
     }
 
+    pub fn is_empty(&self) -> bool {
+        matches!(self, Self::EmptyFile)
+    }
```

Comment:
Since this call is over all of the FileContent this should take into account what it means to be empty for the other types as well. We generally _shouldn't_ have other entries that are empty but that doesn't mean they won't exist.